lib/process-services-cloud/src/lib/process/process-filters/components/process-filters-cloud.component.spec.ts

```diff
@@ -129,88 +129,93 @@ describe('ProcessFiltersCloudComponent', () => {
     });
 
     it('should emit success with the filters when filters are loaded', async () => {
+        const successSpy = spyOn(component.success, 'emit');
         const appName = 'my-app-1';
         const change = new SimpleChange(null, appName, true);
 
         component.ngOnChanges({ 'appName': change });
         fixture.detectChanges();
         await fixture.whenStable();
 
+        expect(successSpy).toHaveBeenCalledWith(mockProcessFilters);
         expect(component.filters).toBeDefined();
         expect(component.filters[0].name).toEqual('FakeAllProcesses');
         expect(component.filters[1].name).toEqual('FakeRunningProcesses');
         expect(component.filters[2].name).toEqual('FakeCompletedProcesses');
     });
 
-    it('should select the first process cloud filter as default', async () => {
+    it('should not select any filter as default', async () => {
         const appName = 'my-app-1';
         const change = new SimpleChange(null, appName, true);
 
         component.ngOnChanges({ 'appName': change });
         fixture.detectChanges();
         await fixture.whenStable();
 
-        expect(component.currentFilter).toBeDefined();
-        expect(component.currentFilter.name).toEqual('FakeAllProcesses');
+        expect(component.currentFilter).toBeUndefined();
     });
 
     it('should select the filter based on the input by name param', async () => {
-        component.filterParam = { name: 'FakeRunningProcesses' };
+        const filterSelectedSpy = spyOn(component.filterSelected, 'emit');
         const appName = 'my-app-1';
         const change = new SimpleChange(null, appName, true);
 
+        component.filterParam = { name: 'FakeRunningProcesses' };
```

Comment:
why are not passing the filter as we should do? through the ngOnChange

Reply:
Fixed, thanks